provisioner/puppet-masterless/provisioner.go

```diff
@@ -73,7 +78,8 @@ func (p *Provisioner) Prepare(raws ...interface{}) error {
 
 	// Set some defaults
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = "{{.FacterVars}} {{if .Sudo}} sudo -E {{end}}" +
+		p.config.ExecuteCommand = "cd {{.WorkingDir}} && " +
```

Comment:
`sudo` would probably be the biggest problem. Not `cd` which is both on Windows and Linux.